src/components/FlatParameters/FlatParameters.js

```diff
@@ -229,7 +231,17 @@ export const FlatParameters = ({
                 return (
                     <TextField
                         fullWidth
-                        sx={{ input: { textAlign: 'right' } }}
+                        sx={{
+                            ...(showSeparator
+                                ? {
+                                      input: {
+                                          textAlign: 'right',
+                                          height: '10px',
+                                      },
+                                      width: '300px',
+                                  }
```

Comment:
can use arrays to merge sx instead of duplicating textAlign: right I think

Reply:
I updated it